lib/widgets/store.dart

```diff
@@ -75,43 +72,6 @@ class GlobalStoreWidget extends StatefulWidget {
     return widget!.store;
   }
 
-  @override
-  State<GlobalStoreWidget> createState() => _GlobalStoreWidgetState();
-}
-
-class _GlobalStoreWidgetState extends State<GlobalStoreWidget> {
-  GlobalStore? store;
-
-  @override
-  void initState() {
-    super.initState();
-    (() async {
-      final store = await ZulipBinding.instance.getGlobalStoreUniquely();
-      setState(() {
-        this.store = store;
-      });
-    })();
-  }
-
-  @override
-  Widget build(BuildContext context) {
-    final store = this.store;
-    if (store == null) return widget.placeholder;
-    return _GlobalStoreInheritedWidget(store: store, child: widget.child);
-  }
-}
-
-// This is separate from [GlobalStoreWidget] only because we need
-// a [StatefulWidget] to get hold of the store, and an [InheritedWidget] to
-// provide it to descendants, and one widget can't be both of those.
-class _GlobalStoreInheritedWidget extends InheritedNotifier<GlobalStore> {
```

Comment:
There's a comment below which refers to this class, so needs to be updated :slightly_smiling_face: